website/docs/reference/dbt-jinja-functions/cross-database-macros.md

```diff
@@ -0,0 +1,537 @@
+---
+title: "cross-database macros"
+id: "cross-database-macros"
+---
+
+# Overview
+
+These macros make it easier for package authors (especially those writing modeling packages) to implement cross-database
+compatibility. In general, you should not need these macros in your own dbt project (unless it is a package or you otherwise desire cross-database compatibility).
+
+## All functions (alphabetical)
+- [dbt.any_value](#any_value)
+- [dbt.bool_or](#bool_or)
+- [dbt.cast_bool_to_text](#cast_bool_to_text)
+- [dbt.concat](#concat)
+- [dbt.current_timestamp](#current_timestamp)
+- [dbt.current_timestamp_in_utc](#current_timestamp_in_utc)
+- [dbt.dateadd](#dateadd)
+- [dbt.datediff](#datediff)
+- [dbt.date_trunc](#date_trunc)
+- [dbt.escape_single_quotes](#escape_single_quotes)
+- [dbt.except](#except)
+- [dbt.hash](#hash)
+- [dbt.intersect](#intersect)
+- [dbt.last_day](#last_day)
+- [dbt.length](#length)
+- [dbt.listagg](#listagg)
+- [dbt.position](#position)
+- [dbt.replace](#replace)
+- [dbt.right](#right)
+- [dbt.safe_cast](#safe_cast)
+- [dbt.split_part](#split_part)
+- [dbt.string_literal](#string_literal)
+
+## Set functions
```

Comment:
Let's add links to the anchors down below for each category. That probably requires changing this header, though, to avoid conflicting anchors:
```suggestion
[**Set functions**](#set-functions)
```